velox/type/Type.cpp

```diff
@@ -1008,6 +1008,7 @@ std::string DateType::toString(int32_t days) const {
       days);
   TimestampToStringOptions options;
   options.mode = TimestampToStringOptions::Mode::kDateOnly;
+  options.zeroPaddingYear = true;
```

Comment:
Can you add a comment for this option? What does it do?

Reply:
It pads the 'year' portion of the string out to four digits, if necessary. I enabled this to ensure years less than `1000` still comply with the four digit `YYYY` format.

Reply:
Added comment.